src/windows/AskingWindow.java

```diff
@@ -0,0 +1,143 @@
+import javax.swing.JTextPane;
+import javax.swing.text.SimpleAttributeSet;
+import javax.swing.text.StyleConstants;
+import javax.swing.text.StyledDocument;
+
+public class AskingWindow extends JPanel{
+    
+     JPanel buttonPanel = new JPanel();
+     JPanel choosenButtonsPanel = new JPanel();
+     JPanel textFieldPanel = new JPanel();
+     JTextPane englishText = new JTextPane();
+     JTextPane hungarianText = new JTextPane();
+     CardDeck cardDeck;
+     
+     private List<Card> cards;
+     private DatabaseHandler databaseHandler;
+    
+    AskingWindow()
+    {  
+        SetPanelsAndButtons();
+        cardDeck = new CardDeck();
+    }
+
+    // set the panels and the buttons
+    private void SetPanelsAndButtons() {
+        buttonPanel.setLayout(new BorderLayout());
+        choosenButtonsPanel.setLayout(new FlowLayout());
+        textFieldPanel.setLayout(new BorderLayout());
+        textFieldPanel.setPreferredSize(new Dimension(300,100));
+        
+        SetTextFieldsAndProperties();
+        
+        this.add(Box.createVerticalStrut(300));
+        textFieldPanel.add(englishText, BorderLayout.NORTH);
+        textFieldPanel.add(hungarianText, BorderLayout.SOUTH);
+        
+        addButton(":(", choosenButtonsPanel);
+        addButton("OK", choosenButtonsPanel);
+        addButton(":)", choosenButtonsPanel);
+        buttonPanel.add(Box.createVerticalStrut(70));
+        buttonPanel.add(textFieldPanel, BorderLayout.NORTH);
+        buttonPanel.add(choosenButtonsPanel, BorderLayout.SOUTH);
+        this.add(buttonPanel);
+    }
+
+    private void SetTextFieldsAndProperties() {
+        englishText.setText("english");
+        hungarianText.setText("hungarian");
+        englishText.setEditable(false);
+        hungarianText.setEditable(false);
+        // set all text to center
+        StyledDocument doceng = englishText.getStyledDocument();
+        StyledDocument dochung = hungarianText.getStyledDocument();
+        SimpleAttributeSet center = new SimpleAttributeSet();
+        StyleConstants.setAlignment(center, StyleConstants.ALIGN_CENTER);
+        doceng.setParagraphAttributes(0, doceng.getLength(), center, false);
+        dochung.setParagraphAttributes(0, dochung.getLength(), center, false);
+    }
+    
+    private static class chooseButtonListener implements ActionListener {
+
+        @Override
+        public void actionPerformed(ActionEvent e) {
+            String buttonText = e.getActionCommand();
+            System.out.println(e.getActionCommand());
+            switch (buttonText) {
+                case "Card Manager":
+                    
+                    break;
+            }
+        }
+        
+    }
+    
+    private static void addButton(String text, Container container){
+        JButton button = new JButton(text);
+        button.addActionListener(new AskingWindow.chooseButtonListener());
+        container.add(button);
+        container.add(Box.createHorizontalStrut(70));
```

Comment:
There is a typo here, that is why no margin is visible between the two text fields. Please rewrite it to `Box.createHorizontalStruct(70)`.